packages/commonwealth/server/routes/profiles/getProfiles.ts

```diff
@@ -62,6 +61,10 @@ const getProfiles = async (
     {
       model: models.Address,
       required: true,
+      include: [
+        { model: models.Thread, as: 'Address' },
```

Comment:
Why are we including the Threads as "Address"?

Reply:
Good point, I had just followed the specification in the issue without giving it too much thought. I will fix it.

Reply:
This is because there are two associations (see thread.ts)

We have:
```    models.Thread.belongsTo(models.Address, {
      as: 'Address',
      foreignKey: 'address_id',
      targetKey: 'id',
    });
```

and 
```
    models.Thread.belongsToMany(models.Address, {
      through: models.Collaboration,
      as: 'collaborators',
    });
```

So the ORM requires you to specify one of them.